pkg/catalog/tags.go

```diff
@@ -66,6 +66,7 @@ var knownsTags = []string{
 	"!!map",
 	"!!str",
 	"!!seq",
+	"!!null",
 
 	// Custom tags
 	"!lock",
```

Comment:
Var should probably be called `knownTags` (while we are at it!)

Reply:
typos! They are everywhere :\